Editor/Modules/TextureModule.cs

```diff
@@ -0,0 +1,89 @@
+using System.Collections.Generic;
+using UnityEngine.Profiling;
+using UnityEditor;
+using UnityEngine;
+
+
+namespace Unity.ProjectAuditor.Editor.Modules
+{
+    public enum TextureProperties
+    {
+        Name,
+        Shape,
+        ImporterType,
+        Format,
+        TextureCompression,
+        MipMapEnabled,
+        Readable,
+        Resolution,
+        SizeOnDisk,
+        Num
+    }
+
+    class TextureModule : ProjectAuditorModule
+    {
+        public static string[] searchTheseFolders;
+        public TextureImporter TextProp;
+
+        private static readonly IssueLayout k_IssueLayout = new IssueLayout
+        {
+            category = IssueCategory.Texture,
+            properties = new[]
+            {
+                new PropertyDefinition {type = PropertyTypeUtil.FromCustom(TextureProperties.Name), format = PropertyFormat.String, name = "Name", longName = "Texture Name" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.Shape), format = PropertyFormat.String, name = "TextureShape", longName = "Texture Shape" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.ImporterType), format = PropertyFormat.String, name = "Importer Type", longName = "Texture Importer Type" },
+                new PropertyDefinition {type = PropertyTypeUtil.FromCustom(TextureProperties.Format), format = PropertyFormat.String, name = "Format", longName = "Texture Format" },
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(TextureProperties.TextureCompression), format = PropertyFormat.String, name = "Compression Used?", longName = "Texture Compression" },
```

Comment:
please rename "Compression Used?" to just "Compression"

Reply:
Renamed.